### PR TITLE
WIP: chore(dependencies): Bump dependencies eslint-config-sentry-react

### PR DESCRIPTION
**Pull request by priscilawebdev**
Bump dependencies eslint-config-sentry-react

### CODE REVIEW

File: packages/eslint-config-sentry-react/package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "url": "https://github.com/getsentry/eslint-config-sentry/issues"
   },
   "peerDependencies": {
-    "eslint": ">=5",
-    "eslint-plugin-import": "~2.22.0",
-    "eslint-plugin-jest": "~22.17.0",
-    "eslint-plugin-react": "~7.15.1"
+    "eslint": "^7.11.0",
```
Comment (billyvg) [inline packages/eslint-config-sentry-react/package.json:22]: I had problems with eslint7 fyi

Comment (priscilawebdev) [reply packages/eslint-config-sentry-react/package.json:22]: please what kind of problems did you have?

Comment (priscilawebdev) [reply packages/eslint-config-sentry-react/package.json:22]: yes we will have to wait a bit more

**Comment by priscilawebdev** [comment]
closing this for now

**Pull request closed without merging**